_posts/2016-05-03-scaling-cto-talk.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout:       post
+title:        The Scaling CTO - Talk from the Wheelhouse CTO Summit at the NASDAQ
```

Comment:
Also could consider shilling up the title a bit:

How to scale your engineering organization? How to scale as a CTO etc...

"Talk from the Wheelhouse CTO Summit" can be subheader or H1 at the top of the post.